packages/bot/src/components/toolbar.jsx

```diff
@@ -184,122 +202,103 @@ const ButtonGroup = ({
                 alignment='bottom'
                 message={localize('Run')}
             >
-                <ToolbarRunIcon className='toolbar__icon' onClick={onRunClick} />
+                <ToolbarRunIcon
+                    className='toolbar__icon'
+                    onClick={onRunClick}
+                />
             </Popover>
         }
         <Popover
             alignment='bottom'
             message={localize('Sort')}
         >
-            <ToolbarReaarangeIcon className='toolbar__icon' onClick={onSortClick} />
+            <ToolbarReaarangeIcon
+                className='toolbar__icon'
+                onClick={() => Blockly.derivWorkspace.cleanUp()}
```

Comment:
Hmm, I moved it here because the logic isn't doing anything related to the store or manipulating any observable, but I agree it makes sense to keep logic in the store rather than here. I'll update it.